src/app/modules/form/admin-form/admin-form.middlewares.ts

```diff
@@ -41,24 +41,6 @@ export const updateSettingsValidator = celebrate({
       gstRegNo: Joi.string().allow(''),
     }),
     payments_field: Joi.object({ gst_enabled: Joi.boolean() }),
-    workflow: Joi.array()
```

Comment:
Likely that it might break for a very very minute set of users that were editing their workflow. Just need to let the team know that this is not a bug!